poppy/matrixDFT.py

```diff
@@ -302,22 +263,10 @@ def DFT_fftstyle_rect(pupil, nlamD, npix, inverse=False):
     t2 = np.dot(t1, expXU)
 
     if inverse:
-        #print expYV.shape, pupil.shape, expXU.shape
         t2 = t2[::-1, ::-1]
-    #else:
-        #expYV = expYV.T.copy()                          # matrix npixY * npupY
-        #t1 = np.dot(expYV, pupil)                       # matrix npixY * npupX?     dot prod combined last axis of expYV and first axis of pupil.
-        #t2 = np.dot(t1, expXU)                          # matrix npixY * npixX
-        #print expYV.shape, pupil.shape, expXU.shape
-
-    #print ""
-
-    #return  nlamD/(npup*npix) *   t2 * dx * dy
     # normalization here is almost certainly wrong:
```

Comment:
Hmmm, although I note that the tests are only for the forward direction transforms. For completeness' sake we should maybe extend that to also test the inverse direction (after sorting out the parity issue from Anand).